src/Tools/dotnet-counters/CounterMonitor.cs

```diff
@@ -842,6 +847,7 @@ private Task<int> Start()
             });
 
             monitorTask.Start();
+            var inputLoopStart = DateTime.Now;
```

Comment:
Could we use a `StopWatch` here? This is prone to clock skew.

Reply:
Can do, I'll update the PR tomorrow

Reply:
I'm not sure what's going on on the CI side of things. Looks like the builds may be stuck?

Reply:
Sorry - I didn't see you'd pushed any changes. I'll check them in a bit. I also kicked off the stuck leg again, it lost connectivity and the reporting is not very resilient to that.